ansible/roles/node/templates/kubelet.j2

```diff
@@ -13,7 +13,9 @@ KUBELET_HOSTNAME="--hostname-override={{ inventory_hostname }}"
 # location of the api-server
 KUBELET_API_SERVER="--api-servers=https://{{ groups['masters'][0] }}:{{ kube_master_api_port }}"
 
+{% if kubelet_options is undefined -%}
```

Comment:
Yes, it will be false if the user specifies it in somewhere like all.yml. We should let the user know that it's an array not string.